Makefile

```
@@ -216,7 +216,7 @@ endif
 	($(SA) $(ENV) && GATEWAY_HOST=$(ITEST_DOCKER_HOST) LOG_LEVEL=$(LOG_LEVEL) KERNEL_USERNAME=$(ITEST_USER) KERNEL_LAUNCH_TIMEOUT=$(ITEST_KERNEL_LAUNCH_TIMEOUT) $(ITEST_DOCKER_KERNELS) ITEST_HOSTNAME_PREFIX=$(ITEST_USER) pytest -v -s $(TEST_DEBUG_OPTS) $(ITEST_DOCKER_TESTS))
 	@echo "Run \`docker service logs itest-docker\` to see enterprise-gateway log."
 
-PREP_TIMEOUT?=60
+PREP_TIMEOUT?=120
```

Per my comment on the PR, let's go ahead and give 180 a shot here.  Should that fail, then perhaps 240, but I think if that too were to fail we'd want to get a better understanding of where the delay is actually occurring.
```suggestion
PREP_TIMEOUT?=180
```